eacl/filter.go

```diff
@@ -174,3 +174,11 @@ func (f *Filter) UnmarshalJSON(data []byte) error {
 
 	return nil
 }
+
+// EqualFilters compares Filter with each other.
+func EqualFilters(f1, f2 Filter) bool {
```

Comment:
Lets for now leave all equal functions except `EqualTables` unexported, I don't think they are useful.